core/src/main/scala/com/salesforce/op/OpWorkflowModelReader.scala

```diff
@@ -154,12 +164,24 @@ class OpWorkflowModelReader(val workflow: OpWorkflow) extends MLReader[OpWorkflo
     }
   }
 
-  private def resolveRawFeatureDistributions(json: JValue): Try[Array[FeatureDistribution]] = {
-    if ((json \ RawFeatureDistributions.entryName) != JNothing) { // for backwards compatibility
-      val distString = (json \ RawFeatureDistributions.entryName).extract[String]
+  private def resolveRawFeatureDistributions(json: JValue): Try[Seq[FeatureDistribution]] = {
+
+    val rawFeatureDistributionsEntryName = "rawFeatureDistributions"
```

Comment:
See other comment

Would prefer leaving as is, based on choice to keep everything needed to maintain backwards compatibility as much as possible in the Reader. Also, I like that `FeatureDistribution` refers to a general feature distribution, not necessarily raw features. But can change if you feel strongly otherwise

Reply:
please add a comment that this is depreciated